tests/lib/config/environments.js

```diff
@@ -17,36 +17,37 @@ const assert = require("chai").assert,
 //------------------------------------------------------------------------------
 
 describe("Environments", () => {
+    const environments = new Environments();
 
     describe("load()", () => {
 
         it("should have all default environments loaded", () => {
             Object.keys(envs).forEach(envName => {
-                assert.deepEqual(Environments.get(envName), envs[envName]);
+                assert.deepEqual(environments.get(envName), envs[envName]);
             });
         });
 
         it("should have all default environments loaded after being cleared", () => {
-            Environments.testReset();
+            environments.testReset();
 
             Object.keys(envs).forEach(envName => {
-                assert.deepEqual(Environments.get(envName), envs[envName]);
+                assert.deepEqual(environments.get(envName), envs[envName]);
             });
         });
     });
 
     describe("define()", () => {
 
         afterEach(() => {
-            Environments.testReset();
```

Comment:
Instead of resetting the environment after each test run, can we just use a different `Environment` instance each time? That way we can get rid of `testReset`.